app/models/miq_worker/container_common.rb

```diff
@@ -44,15 +44,28 @@ def default_image
     def resource_constraints
       mem_threshold = self.class.worker_settings[:memory_threshold]
       cpu_threshold = self.class.worker_settings[:cpu_threshold_percent]
+      mem_default   = self.class.worker_settings[:memory_minimum]
+      cpu_default   = self.class.worker_settings[:cpu_minimum_percent]
```

Comment:
Should we have defaults in the settings.yml like we do for the [others](https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml#L1080-L1084)?

Note, this setting is pods only whereas the prior ones are not, so it might be weird to see these settings and only honor them in container based workers.

Reply:
> Note, this setting is pods only whereas the prior ones are not, so it might be weird to see these settings and only honor them in container based workers.

I think we should honor them in systemd eventually as well but we can do it in follow ups.  In the original issue I mentioned this.

Reply:
I'm not sure if this concept translates to systemd. Do they have minimum memory to start a service kind fo thing?

It probably applies to right scaling.

Reply:
ah yeah good point...they have maximums, but not necessarily minimums... perhaps @agrare knows.

Reply:
There are MemoryLow and MemoryMin values, not sure they match exactly what k8s does it sounds like it just leaves pages available for the service above the min threshold so they aren't "stolen" by other processes.

https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#memory-interface-files

I don't believe this means the service won't be started unless that minimum amount of memory is available

Reply:
> can't find where `@carbonin` said this originall

Found it - `@carbonin`'s comment in #20193

> For the rest of the components in github.com/manageiq/manageiq-pods I don't set a request by default so that the containers will be schedule-able out of the box even on small environments like minikube.

Reply:
haha, the same discussion is down below... I mentioned we can honor 0/nil value as a no minimum limit.